script.js

```diff
@@ -54,6 +54,36 @@ form.addEventListener('submit', (event) => {
     error.textContent = '';
     error.classList.add('hidde-span');
   }
+  localStorage.clear();
+});
+
+let userData = {};
+function saveData(user) {
+  const fullNameForm = document.getElementById('full-name').value;
+  const emailForm = document.getElementById('email').value;
+  const messageForm = document.getElementById('message').value;
+  userData = {
+    'full-name': fullNameForm,
+    email: emailForm,
+    message: messageForm,
+  };
+  localStorage.setItem('user', JSON.stringify(user));
+}
+
+form.addEventListener('change', () => {
+  saveData(userData);
+});
```

Comment:
- [ ] Following the project requirement,  all entered values (Name, Email, Message) should be saved in the local storage after filling the form, **and not submitting it**. If the user reloads the page or closes the browser and opens it again, the form inputs should be auto-populated with the values entered in the previous visit to the page.
- *Please notice that your code doesn't save all the data in the localStorage. Kindly work on that 😃.*

Reply:
Addressed 🔥